f8a_worker/dispatcher/nodes.yml

```diff
@@ -477,7 +477,7 @@
       import: 'f8a_worker.storages'
       configuration:
         <<: *configurationS3
-        bucket_name: '{DEPLOYMENT_PREFIX}-bayesian-user-intent'
+        bucket_name: '{DEPLOYMENT_PREFIX}-manual-tags'
```

Comment:
Do we want to reuse the old bucket? Wouldn't it be better to create a new adapter for this purpose?

Reply:
this is a bucket which we are not using anymore, so i updated it to use in this crowd sourcing work flow.